app/models/etymology_type_association.rb

```diff
@@ -0,0 +1,9 @@
+class EtymologyTypeAssociation < EtymologySubjectAssociation
+  before_save :default_values
```

Comment:
I think this is unnecessary, instead of adding default before saving, default should be added upon creation of object, which it is with the default scope.